src/esp/sim/Simulator.cpp

```diff
@@ -713,6 +713,44 @@ bool Simulator::isNavMeshVisualizationActive() {
   return (navMeshVisNode_ != nullptr && navMeshVisPrimID_ != ID_UNDEFINED);
 }
 
+int Simulator::showTrajectoryVisualization(const std::string& assetName,
```

Comment:
This probably over kill. placing one tube at a time is probably sufficient.

Reply:
Come on ;) Issuing 1000 separate draw calls for a path visualization isn't going to be fast.

Reply:
Support for multiple tubes at once makes sense IMO.